### DISCUSSION TITLE
Pyraf Warning: 741 hlib$motd

### DISCUSSION THREAD

**Discussion by pjpessi** [iraf-community/iraf#224] [Q&A]
Hi all, I have an IRAF and pyraf installation on my debian 11 OS. 
Everything seems to work ok but I get a 
`Warning: 741 hlib$motd`
everytime I initialize pyraf in the python 2.7 environment.
Does anyone know what this warning is about?
Thanks

**Comment by olebole** [comment]
Code 741 means "cannot open file", and why this happens depends on which versions of IRAF and PyRAF you have installed.
Debian 11 comes with IRAF 2.16.1+2018.11.01, but not with a Python 2.7 compatible PyRAF.

**Reply by pjpessi** [reply]
I made an python 2.7 environment and installed pyraf using conda, I thought it was supposed to work like that

**Reply by olebole** [reply]
The suggested way to install IRAF and PyRAF on Debian 11 is 
```shell
$ sudo apt install iraf python3-pyraf
```
If you install PyRAF from other sources, you should make sure that the dependencies are resolved properly and the environment variables are set to the correct paths. You then need to have an IRAF version installed then and point the **$iraf** environment variable to the installation path.

**Reply by pjpessi** [reply]
Thanks a lot, in the end I had a PATH issue, I solved it installing things manually. Cheers
